docs/getting-started/integrate/node.mdx

```diff
@@ -9,6 +9,34 @@ One of the most common usages of Mock Service Worker in Node is utilizing our re
 
 It's recommended to configure API mocking as a part of your tests teardown, so that your tests don't have to reference any mocking during their runs, focusing on testing what matters.
 
+## Configure server
```

Comment:
Could you please move this section ("Configure server") before the "Setup" section?
I think the logical flow is that:

1. You create the `src/mocks/server.js` definition.
1. You _may_ use it in tests (what this page illustrates).

But the first step is the same regardless of where you aim to use request interception. Does this make sense to you?

Reply:
Sounds good. I put it there because that's the order in the browser instructions (Setup > Configure) but I did question the flow.

Reply:
Yeah, I see you did a great job in following the order/phrasing as in the other sections of the tutorial 👍 However, I think we can safely re-order them so it would make the most sense to the reader. Thanks for the update!